package.json

```diff
@@ -19,9 +19,9 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bson": "^4.7.0",
+    "bson": "^5.0.0",
     "kareem": "2.5.1",
-    "mongodb": "4.13.0",
+    "mongodb": "5.0.0",
```

Comment:
Here fixed version is used but 5.0 has some critical issues like this:
https://jira.mongodb.org/projects/NODE/issues/NODE-5052?filter=allopenissues

So we should wait for next driver release.